src/test/java/uk/nhs/tis/trainee/notifications/event/LtftListenerIntegrationTest.java

```diff
@@ -214,21 +221,29 @@ void shouldSendDefaultNotificationsWhenTemplateVariablesNull(String state, Notif
         .pollInterval(Duration.ofSeconds(2))
         .atMost(Duration.ofSeconds(10))
         .ignoreExceptions()
-        .untilAsserted(() -> verify(mailSender).send(messageCaptor.capture()));
-
-    MimeMessage message = messageCaptor.getValue();
-    Document content = Jsoup.parse((String) message.getContent());
+        .untilAsserted(() -> verify(mailSender, times(expectedNotificationEmailCount(state)))
```

Comment:
This is where things start going pear shaped (not specific to this test, in all of them).
When more than one email is expected, sometimes this collects two of the trainee version, sometimes two of the TPD version, and sometimes the expected one of each - although the EmailService is sending the correct mails afaik.